src/payment/strategies/cybersource/cybersource-threedsecure-payment-processor.spec.ts

```diff
@@ -0,0 +1,356 @@
+import { createClient as createPaymentClient } from '@bigcommerce/bigpay-client';
+import {createAction, createErrorAction, Action} from '@bigcommerce/data-store';
+import { createRequestSender } from '@bigcommerce/request-sender';
+import { createScriptLoader } from '@bigcommerce/script-loader';
+import {of, Observable} from 'rxjs/index';
+
+import { getCartState } from '../../../cart/carts.mock';
+import {
+    createCheckoutStore,
+    CheckoutRequestSender,
+    CheckoutStore,
+    CheckoutValidator
+} from '../../../checkout';
+import { getCheckoutState } from '../../../checkout/checkouts.mock';
+import { RequestError, StandardError } from '../../../common/error/errors';
+import MissingDataError from '../../../common/error/errors/missing-data-error';
+import NotInitializedError from '../../../common/error/errors/not-initialized-error';
+import { getResponse } from '../../../common/http-request/responses.mock';
+import { getConfigState } from '../../../config/configs.mock';
+import { getCustomerState } from '../../../customer/customers.mock';
+import { OrderActionCreator, OrderActionType, OrderRequestSender } from '../../../order';
```

Comment:
If you import more than 3 modules please separate in a new line. 💅
```suggestion
import {
  OrderActionCreator,
  OrderActionType,
  OrderRequestSender
} from '../../../order';
```